x/employeestore/internal/types/msgs.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const RouterKey  = ModuleName
+
+type MsgSetInfo struct {
+	EmployeeId int64 `json:"employee_id"`
+	EmployeeName string `json:"employee_name"`
+}
+
+func NewMsgSetInfo (EmployeeName string) MsgSetInfo{
```

Comment:
EmployeeInfo should be a struct with name as part of it. This will make it easy to extend